views/cypress/tests/properties/simple.spec.js

```diff
@@ -123,20 +124,32 @@
             options.propertyType = propertiesInfo.singleSearchList.type;
             cy.addPropertyToClass(options);
         });
+
+        it('can edit and add new "Calendar" type property to the item class', function () {
+            options.propertyName = propertiesInfo.calendar.name;
+            options.propertyType = propertiesInfo.calendar.type;
+            cy.addPropertyToClass(options);
+        });
+
+        it('can edit and add new "Password" type property to the item class', function () {
+            options.propertyName = propertiesInfo.password.name;
+            options.propertyType = propertiesInfo.password.type;
```

Comment:
I know it is not the case now, but should not we avoid using the same `option` variable reference for all the tests, since it can be changed and affect the test behavior.